flink-metrics/flink-metrics-graphite/pom.xml

```diff
@@ -65,31 +65,4 @@ under the License.
 			<version>${dropwizard.version}</version>
 		</dependency>
 	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-shade-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>shade-flink</id>
-						<phase>package</phase>
-						<goals>
-							<goal>shade</goal>
-						</goals>
-						<configuration>
-							<artifactSet>
-								<includes>
-									<include>org.apache.flink:flink-metrics-dropwizard</include>
-									<include>io.dropwizard.metrics:metrics-core</include>
```

Comment:
ah, hold on, we do need this. Otherwise we aren't building a fat jar. Were we not relocating these? That's odd 😕

Reply:
You mean we still need to shade "io.dropwizard.metrics:metrics-core" and "io.dropwizard.metrics:metrics-graphite", right?

Reply:
So this is where terminology is the name of the game. Forget the term "shading", because it is quite ambiguous.

We do not need to _relocate_ any dependency, but we still want to _bundle_ them in the jar.